sdk/storage/Azure.Storage.DataMovement.Files.Shares/tests/Shared/DisposingShare.cs

```diff
@@ -26,6 +26,17 @@ public static async Task<DisposingShare> CreateAsync(ShareClient share, IDiction
             return new DisposingShare(share);
         }
 
+        public static async Task<DisposingShare> CreateAsyncNFS(ShareClient share, IDictionary<string, string> metadata)
```

Comment:
nit: This should probably be `CreateNFSAsync`. Async should always be at the end.